modules/olm-operator-lifecycle-manager.adoc

```diff
@@ -179,38 +181,421 @@ required to update a CSV to the latest version in a channel (stepping through
+
+* If more than one OperatorGroup exists in a single namespace, any CSV created
+in that namespace will transition to a failure state with the reason
+`TooManyOperatorGroups`. CSVs in a failed state for this reason will
+transition to pending once the number of OperatorGroups in their namespaces
+reaches one.
+* If a CSV's InstallModes do not support the target namespace selection of the
+OperatorGroup in its namespace, the CSV will transition to a failure state
+with the reason `UnsupportedOperatorGroup`. CSVs in a failed state for this
+reason will transition to pending once either the OperatorGroup's target
+namespace selection changes to a supported configuration, or the CSV's
+InstallModes are modified to support the OperatorGroup's target namespace
+selection.
+
+[discrete]
+=== Target namespace selection
+
+Specify the set of namespaces for the OperatorGroup using a label
+selector with the `spec.selector` field:
+
+[source,yaml]
+----
+apiVersion: operators.coreos.com/v1alpha2
+kind: OperatorGroup
+metadata:
+  name: my-group
+  namespace: my-namespace
+spec:
+  selector:
+    cool.io/prod: "true"
+----
+
+You can also explicitly name the target namespaces using the
+`spec.targetNamespaces` field:
+
+[source,yaml]
+----
+apiVersion: operators.coreos.com/v1alpha2
+kind: OperatorGroup
+metadata:
+  name: my-group
+  namespace: my-namespace
+  spec:
+    targetNamespaces:
+    - my-namespace
+    - my-other-namespace
+    - my-other-other-namespace
+----
+
+NOTE: If both `spec.targetNamespaces` and `spec.selector` are defined,
+`spec.selector` is ignored.
+
+Alternatively, you can omit both `spec.selector` and `spec.targetNamespaces` to
+specify a _global_ OperatorGroup, which selects all namespaces:
+
+[source,yaml]
+----
+apiVersion: operators.coreos.com/v1alpha2
+kind: OperatorGroup
+metadata:
+  name: my-group
+  namespace: my-namespace
+----
+
+The resolved set of selected namespaces is shown in an OperatorGroup's
+`status.namespaces` field. A global OperatorGroup's `status.namespace` contains
+the empty string (`""`), which signals to a consuming Operator that it should
+watch all namespaces.
+
+[discrete]
+=== OperatorGroup CSV annotations
+
+Member CSVs of an OperatorGroup have the following annotations:
+
+[cols="1,1",options="header"]
+|===
+|Annotation |Description
+
+|`olm.operatorGroup=<group_name>`
+|Contains the name of the OperatorGroup.
+
+|`olm.operatorGroupNamespace=<group_namespace>`
+|Contains the namespace of the OperatorGroup.
+
+|`olm.targetNamespaces=<target_namespaces>`
+|Contains a comma-delimited string listing the OperatorGroup's target namespace
+ selection.
+|===
+
+NOTE: All annotations except `olm.targetNamespaces` are included with copied
+CSVs. Omitting the `olm.targetNamespaces` annotation on copied CSVs prevents the
+names of target namespaces from being leaked between tenants.
+
+[discrete]
+=== Provided APIs annotation
+
+Information about what `GroupVersionKinds` (GVKs) are provided by an
```

Comment:
At this point, I don't know what the `GroupVersionKinds` or the `olm.providedAPIs` annotation is for.